crates/networking/gossipsub/src/topics.rs

```diff
@@ -0,0 +1,99 @@
+use libp2p::gossipsub::IdentTopic as Topic;
+
+pub const TOPIC_PREFIX: &str = "eth2";
+pub const ENCODING_POSTFIX: &str = "ssz_snappy";
+pub const BEACON_BLOCK_TOPIC: &str = "beacon_block";
+pub const BEACON_AGGREGATE_AND_PROOF_TOPIC: &str = "beacon_aggregate_and_proof";
+pub const VOLUNTARY_EXIT_TOPIC: &str = "voluntary_exit";
+pub const PROPOSER_SLASHING_TOPIC: &str = "proposer_slashing";
+pub const ATTESTER_SLASHING_TOPIC: &str = "attester_slashing";
+pub const BEACON_ATTESTATION_PREFIX: &str = "beacon_attestation_";
+pub const SYNC_COMMITTEE_PREFIX_TOPIC: &str = "sync_committee_";
+pub const SYNC_COMMITTEE_CONTRIBUTION_AND_PROOF_TOPIC: &str =
+    "sync_committee_contribution_and_proof";
+pub const BLS_TO_EXECUTION_CHANGE_TOPIC: &str = "bls_to_execution_change";
+pub const LIGHT_CLIENT_FINALITY_UPDATE_TOPIC: &str = "light_client_finality_update";
+pub const LIGHT_CLIENT_OPTIMISTIC_UPDATE_TOPIC: &str = "light_client_optimistic_update";
+pub const BLOB_SIDECAR_PREFIX_TOPIC: &str = "blob_sidecar_";
+
+#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq)]
+pub struct GossipTopic {
+    pub fork: [u8; 4],
+    pub kind: GossipTopicKind,
+}
+
+impl std::fmt::Display for GossipTopic {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(
+            f,
+            "/{}/{}/{}/{}",
+            TOPIC_PREFIX,
+            hex::encode(self.fork),
```

Comment:
```suggestion
            self.fork,
```
if we use B32 we don't need hex encode